packages/@ama-sdk/core/src/plugins/bot-protection-fingerprint/bot-protection-fingerprint.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('BotProtectionFingerprint', () => {
   describe('Retrievers', () => {
 
     describe('impervaProtectionRetrieverFactory', () => {
-      const consoleMock = jest.spyOn(console, 'warn');
+      const consoleMock = jest.spyOn(console, 'error');
```

Comment:
```suggestion
      const consoleMock = jest.spyOn(console, 'error').mockImplementation();
```
to avoid logging errors in the logs of the test